docs/HowToSupportTizenRtProtectedBuild.md

```diff
@@ -0,0 +1,250 @@
+# Steps for Supporting Protected Build in a new chipset
+
+## Introduction
+This guide explains the changes to be made to support Protected Build for a new Chipset. The user can refer to [Protected Build Guide](ProtectedBuildGuide.md) for detailed descriptions of the steps involved.
+
+### Prerequisites
+- TizenRT flat build needs to be successfully ported and tested on the target CHIPSET board.
```

Comment:
Do we need to explain what flat build is?

Reply:
The most important thing I think is that we must let reader understand clearly.

Is it better to link basic porting guide like the doc of how to add new board and to say it is the flat build?

Reply:
Currently there is no document which explains the meaning of flat build. 
I think we can do as follows:
- Add a section in ProtectedBuildGuide.md to give brief description of flat build before explaining protected build
- Add a link to the docs/Porting_Guide.md and mention that all steps mentioned in that document is for flat build, where as the current document is for protected build.

Reply:
But you add this doc into porting guide. and porting guide includes adding bsp as well as network and file system. Does porting protected build require them as well?
Is it difficult to say that porting flat build is adding bsp (https://github.com/Samsung/TizenRT/blob/master/docs/HowToAddnewBoard.md)?

Reply:
Ok.. I agree. We will add link to HowToAddnewBoard.md and mention that it refers to flat build.

Reply:
Could you share when you will add it?

Reply:
The link is already added in docs/ProtectedBuildGuide.md line number 20.